README.md

```diff
@@ -114,14 +114,19 @@ https://github.com/aloknsingh/ds_xgboost_clf_4_imbalance_data/tree/master/notebo
 ### 4. Upload data
 
 * Return to the project dashboard view and select the `Assets` tab.
-* This project has 1 dataset, which you can upload as a data asset in your project. Do this by loading the dataset into the pop up section on the right hand side. Please see a screenshot of what it should look like below.  
+* This project has 1 dataset (`/data/bank.csv`), which you can upload as a data asset in your project. Do this by loading the dataset into the pop up section on the right hand side. Please see a screenshot of what it should look like below.  
 
 ![](doc/source/images/project-assets.png)
 
 * Once complete, go into your notebook in the edit mode (click on the pencil icon next to your notebook on the dashboard). 
-* Click on the `1001` data icon in the top right. The data files should show up. 
-* Click on each and select `Insert Pandas Data Frame`. Once you do that, a whole bunch of code will show up in your first cell. 
-* Make sure your `bank.csv` is saved as `data_raw_all` so that it is consistent with the original notebook. You may have to edit this because when your data is loaded into the notebook, it may be defined as a continuation of data frames, based on where I left off. This means your data may show up with `bank.csv` as `df_data_2` and so on. Either adjust the data frame names to be in sync with mine (remove where I loaded data and rename your data frames or input your loading information into the original code) or edit the following code below accordingly. Do this to make sure the code will run!
+* Execute the first initial cells until you get to the first cell in the `Data Exploration` section.
+* To load the data asset into the notebook, you first need to clear out the cell of all code except for the last 2 lines. Then place the cursor at the top of the cell above the 2 remaining lines of code.  
+* Click on the `1001` data icon in the top right. The `bank.csv` data file should show up.
+* Click on it and select `Insert Pandas Data Frame`. Once you do that, a whole bunch of code will show up in your cell.
+* Remove the last 2 lines of the inserted code. These lines start with `df_data_1` and will cause errors if left in.
```

Comment:
I think we should also mention this
" After removing above two lines, please add follow lines
`
data_raw_all = pd.read_csv(body, header=0, sep=";")

data_raw_all.head()
`

The attach image shows the details

Reply:
tried to make it clearer... please re-check